app/src/renderer/vuex/modules/distribution.js

```diff
@@ -0,0 +1,178 @@
+import * as Sentry from "@sentry/browser"
+import Vue from "vue"
+import { coinsToObject } from "scripts/common.js"
+
+export default ({ node }) => {
+    const emptyState = {
+        loading: false,
+        loaded: false,
+        error: null,
+        /* totalRewards use the following format:
+            { 
+                denom1: amount1,
+                ... ,
+                denomN: amountN 
+            }
+        */
+        totalRewards: {},
+        /* rewards use the following format:
+            { 
+                validatorAddr1: { 
+                    denom1: amount1,
+                    ... ,
+                    denomN: amountN 
+                },
+                ... ,
+                validatorAddrN: { 
+                    denom1: amount1,
+                    ... ,
+                    denomN: amountN 
+                } 
+            } 
+        */
+        rewards: {},
+        parameters: {},
+        /* outstandingRewards use the following format:
+            { 
+                denom1: amount1,
+                ... ,
+                denomN: amountN 
+            }
+        */
+        outstandingRewards: {}
+    }
+    const state = JSON.parse(JSON.stringify(emptyState))
+
+    const mutations = {
+        setTotalRewards(state, rewards) {
+            state.totalRewards = rewards
+        },
+        setDelegationRewards(state, { validatorAddr, rewards }) {
+            Vue.set(state.rewards, validatorAddr, rewards)
+        },
+        setDistributionParameters(state, parameters) {
+            state.parameters = parameters
+        },
+        setOutstandingRewards(state, outstandingRewards) {
+            state.outstandingRewards = outstandingRewards
+        },
+    }
+    const actions = {
+        reconnected({ state, dispatch, rootState: { session } }) {
+            if (state.loading && session.signedIn) {
+                dispatch(`getTotalRewards`)
+            }
+        },
+        resetSessionData({ rootState }) {
+            rootState.delegation = JSON.parse(JSON.stringify(emptyState))
+        },
+        async getTotalRewards(
+            { state, rootState: { session }, commit }
+        ) {
+            state.loading = true
+            try {
+                const rewardsArray = await node.getDelegatorRewards(session.address)
+                const rewards = coinsToObject(rewardsArray)
+                commit(`setTotalRewards`, rewards)
+                state.error = null
+            } catch (error) {
+                commit(`notifyError`, {
```

Comment:
no more notify

Reply:
use https://github.com/cosmos/voyager/pull/2049/files#diff-b37c59a991d76e68f3b9a84431a39ab5R9 instead

Reply:
Agree, but I'd also propose to leave notifications for `devMode` only. I like to see if something fails when I'm coding. Thoughts @faboweb @sabau ?

Reply:
I'll delete them here but maybe we can update all of them in a later refactor

Reply:
> I like to see if something fails when I'm coding.

we need to display the error message and corresponding component in the UI, just not as a notify.